spec/integration/cli_view_spec.rb

```diff
@@ -24,5 +24,37 @@
       expect(File.exist?(File.join(output_dir, "libraries", "awesomelib.html"))).to be(true)
       expect(File.exist?(File.join(output_dir, "libraries", "newlib.html"))).to be(true)
     end
+
+    context "inqlude-all.json" do
+      it "generates view" do
+        output_dir = given_directory
+        result = run_command(args: ["view", "--output-dir=#{output_dir}"])
```

Comment:
I would suggest to move the `run_command` call to a `before(:all)` block in this context, as it's the same for all the tests here and the data is only read, not modified, so it can be shared between tests. It is a costly call, so better avoid calling it too often.